frontend/src/app/contact/page.tsx

```
@@ -0,0 +1,79 @@
+          <div className=" flex flex-col items-center h-full w-full rounded-md px-6 md:px-10 py-5 main-theme">
+            <span>
+              <h1 className={`${kelly.className} text-4xl md:text-5xl`}>
+                Contact Us <br />
+              </h1>
+            </span>
+            <div className="mt-5 w-full">
+              <form className="">
+                <div>
+                  <label className="label">
+                    <span className="label-text">Email</span>
+                  </label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)